doc/how-to/sql/improving_mysql.rst

```diff
@@ -1,275 +1,258 @@
+        binlog_format = ROW
+        server_id = 1
+        log-bin = mysql-bin
+        interactive_timeout = 3600
+        wait_timeout = 3600
+        max_allowed_packet = 32M
+        socket = /var/lib/mysql/mysql.sock
+        bind-address = 127.0.0.1
 
-#. After exiting nano, we’ll restart mysqld:
+        [client]
+        socket = /var/lib/mysql/mysql.sock
 
-   .. code-block:: bash
+#.  After exiting nano, restart mysqld:
 
-      systemctl restart mysqld
+    ..  code-block:: console
 
-#. Next, let’s install Tarantool and set up spaces for replication.
-   Go to the `Download page
-   <https://www.tarantool.io/en/download/os-installation/rhel-centos/>`_ and
-   follow the instructions there to install Tarantool.
+        $ systemctl restart mysqld
 
-#. Now we will write a standard Tarantool program by editing the Lua example,
-   which comes with Tarantool:
+..  _improving_mysql-setup-tarantool:
 
-   .. code-block:: bash
+Installing and configuring Tarantool
+------------------------------------
 
-      cd
-      nano /etc/tarantool/instances.available/example.lua
+Let’s install Tarantool and set up spaces for replication.
 
-#. Replace the entire contents of the file with the following:
+#.  Go to the `Download page <https://www.tarantool.io/en/download/os-installation/rhel-centos/>`_ and
+    follow the installation instructions.
 
-   .. code-block:: lua
+#.  Install the :ref:`tt CLI <tt-installation>` utility.
 
-      box.cfg {
-          listen = 3301;
-          memtx_memory = 128 * 1024 * 1024; -- 128Mb
-          memtx_min_tuple_size = 16;
-          memtx_max_tuple_size = 128 * 1024 * 1024; -- 128Mb
-          vinyl_memory = 128 * 1024 * 1024; -- 128Mb
-          vinyl_cache = 128 * 1024 * 1024; -- 128Mb
-          vinyl_max_tuple_size = 128 * 1024 * 1024; -- 128Mb
-          vinyl_write_threads = 2;
-          wal_mode = "none";
-          wal_max_size = 256 * 1024 * 1024;
-          checkpoint_interval = 60 * 60; -- one hour
-          checkpoint_count = 6;
-          force_recovery = true;
+#.  Create a tt environment in the current directory using the :ref:`tt init <tt-init>` command.
 
-           -- 1 – SYSERROR
-           -- 2 – ERROR
-           -- 3 – CRITICAL
-           -- 4 – WARNING
-           -- 5 – INFO
-           -- 6 – VERBOSE
-           -- 7 – DEBUG
-           log_level = 7;
-           too_long_threshold = 0.5;
-       }
+#.  Edit the Lua example, which comes with Tarantool:
 
-      box.schema.user.grant('guest','read,write,execute','universe')
+    ..  code-block:: console
 
-      local function bootstrap()
+        $ cd
+        $ nano /etc/tarantool/instances.available/example.lua
 
-          if not box.space.mysqldaemon then
-              s = box.schema.space.create('mysqldaemon')
-              s:create_index('primary',
-              {type = 'tree', parts = {1, 'unsigned'}, if_not_exists = true})
-          end
+#.  Replace the entire contents of the file with the following:
```

Comment:
I guess we don't need the example anymore: there is nothing common with out resulting code. Let's just create the file w/o copying the example.